Sources/KlaviyoUI/KlaviyoWebView/KlaviyoWebViewController.swift

```diff
@@ -91,6 +91,10 @@ class KlaviyoWebViewController: UIViewController, WKUIDelegate {
         }
     }
 
+    func webViewDidClose(_ webView: WKWebView) {
```

Comment:
So this is called by js?

Reply:
out of the box, WKWebView will listen for a `window.close()` JS event. We can respond to the close event by implementing this delegate method